docs/wiki/Testnet-Testing.md

```diff
@@ -1,25 +1,22 @@
 # Testnet Testing
 
-- [Prepare Wallets](#prepare-wallets)
-  - [Run Bitcoin Daemon](#run-bitcoin-daemon)
-  - [Create the Bitcoin Testnet Wallet](#create-the-bitcoin-testnet-wallet)
-  - [Create a Bitcoin Exchange Wallet Configuration File](#create-a-bitcoin-exchange-wallet-configuration-file)
-  - [Run Decred Daemon and Wallet Software](#run-decred-daemon-and-wallet-software)
-  - [Create a Decred Exchange Wallet Configuration File](#create-a-decred-exchange-wallet-configuration-file)
-- [Get the Testnet Server TLS Certificate](#get-the-testnet-server-tls-certificate)
-- [Start the Client](#start-the-client)
-- [Register the Client](#register-the-client)
-- [Acquire Testnet Funds](#acquire-testnet-funds)
-
```

Comment:
Removing because it's not worth the upkeep? I like these. Can it be auto-generated somehow.

Reply:
Not sure about auto generating them but we can keep them if you insist. Other pages don’t have them either.

Reply:
Done